### PR TITLE
update s2i builder image creation docs

### LINKED ISSUE #19783: [question] How to use origin to build custom s2i builder image
What is the idiomatic approach for building a custom S2I builder image in Origin?

##### Version
oc v3.9.0+191fece
kubernetes v1.9.1+a0ce1bc657
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://192.168.42.30:8443
openshift v3.9.0+46ff3a0-18
kubernetes v1.9.1+a0ce1bc657

##### Steps To Reproduce
1. stuck here - while I can use s2i locally and push to Docker Hub, create an ImageStream and then use it ... but I would like to use OpenShift CI/CD features for the builder image itself.

##### Current Result
Creating a customStrategy BuildConfig that references openshift/origin-image-builder causes:
```
error: build error: image "openshift/origin-sti-builder:v3.9.0" must specify a user that is numeric and within the range of allowed users 
```
This did not work (note that I tried several variants):
```yaml
kind: BuildConfig
apiVersion: "v1"
metadata:
  name: s2i-angular-container-build
triggers:
  - type: ConfigChange
  - github:
      secret: _some secret_
    type: GitHub
spec:
  source:
    type: Git
    git:
      uri: https://github.com/_my-git-repo.git_
  strategy:
    type: Source
    sourceStrategy:
      from:
        kind: DockerImage
        name: openshift/origin-sti-builder:v3.9.0
      exposeDockerSocket: true
  output:
    to:
      kind: ImageStreamTag
      name: s2i-angular-container:latest
```
Note that I translated the name sti-image-builder (doesn't seem to exist anymore) to origin-sti-builder. That is probably a leap that does not make sense.

The docs at [Using OpenShift Origin Build for Automated Testing](https://docs.openshift.org/3.9/creating_images/s2i_testing.html#using-openshift-build-for-automated-testing) at least hints that openshift/sti-image-builder should be possible but that image cannot be resolved.

##### Expected Result
Custom S2I builder image can be built within OpenShift Origin.

##### Additional Information

The documentation seems to be out of sync, but I am certain I am doing something wrong.

_What is the idiomatic approach to build an S2I builder image in OpenShift Origin 3.9?_

### PR DESCRIPTION
**Pull request by bparees**
fixes https://github.com/openshift/origin/issues/19783

### CODE REVIEW

File: creating_images/s2i_testing.adoc [hunk 1]
```diff
@@ -143,72 +143,20 @@ Executing these steps is generally enough to tell if the builder image is
...
 be triggered each time you push into your S2I builder image GitHub repository.
 See xref:../dev_guide/builds/triggering_builds.adoc#webhook-triggers[webhook triggers] for more
 information.
 
-You can also use the `*CustomBuild*` to trigger a rebuild of your application
-based on the S2I image you updated. See xref:../dev_guide/builds/triggering_builds.adoc#image-change-triggers[image change triggers]
+You can also use the `*ImageChangeTrigger*` to trigger a rebuild of your applications that are
+based on the S2I builder image you updated. See xref:../dev_guide/builds/triggering_builds.adoc#image-change-triggers[image change triggers]
 for more information.
 endif::openshift-online[]
```
[the remark this answers is not in the document]
Comment (bmcelvee) [reply creating_images/s2i_testing.adoc]: "This tag" being: endif::openshift-online[]

[the remark this answers is not in the document]
Comment (bparees) [reply creating_images/s2i_testing.adoc]: it applies since you can't do docker strategy builds in online.

[the remark this answers is not in the document]
Comment (bmcelvee) [reply creating_images/s2i_testing.adoc]: Got it, thanks!

**Pull request merged**